### PR TITLE
smtp_test.go: Add test for email validity

### PR DESCRIPTION
**Pull request by sladyn98**
Added tests for email validity check.

### CODE REVIEW

File: smtp_test.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package glutton 
+
+import (
+	"testing"
+)
+
```
Comment (glaslos) [inline smtp_test.go]: use gofmt. I recommend using an IDE like VSCode, Atom or GoLand

File: smtp_test.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+func TestValidateMail(t *testing.T) {
+	 if validateMail("MAIL FROM:<gunnerforlife00@gmail.com>")!=true{
+		t.Errorf("Validate email regex failed")
```
Comment (glaslos) [inline smtp_test.go]: use `t.Fatal()`

File: smtp_test.go [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+	 }
+
+	 if validateMail("MAIL FROM:<sladynnunes.com>")!=false{
+		 t.Errorf("Validate email regex failed")
```
Comment (glaslos) [inline smtp_test.go]: did the regex fail or was the input wrong?

File: smtp_test.go (hunk 1 above)
Comment (sladyn98) [reply smtp_test.go]: Yeah I do use VSCode, planning to shift to golang soon

File: smtp_test.go [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package glutton
+
+import (
+	"testing"
+)
+
+func TestValidateMail(t *testing.T) {
+
+	if validateMail("MAIL FROM:<gunnerforlife00@gmail.com>") != true {
```
Comment (glaslos) [inline smtp_test.go]: I think we should use something more generic
```suggestion
	if validateMail("MAIL FROM:<example@example.com>") != true {
```

File: smtp_test.go [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+		t.Fatal("Validate email regex failed")
+	}
+
+	if validateMail("MAIL FROM:<sladynnunes.com>") == true {
```
Comment (glaslos) [inline smtp_test.go]:
```suggestion
	if validateMail("MAIL FROM:<example.com>") == true {
```

File: smtp_test.go (hunk 4 above)
Comment (sladyn98) [reply smtp_test.go]: Yeah definitely I just suggested it for the change I will make the PR tomorrow. I currently don't have my laptop.

**Comment by glaslos** [comment]
For the next PR, don't use force push, just commit your changes.

**Pull request merged**